docs/source/terms_and_model.rst

```diff
@@ -1215,6 +1185,196 @@ order. See :ref:`computed-identifiers` for more information.
 
 
 
+.. _systemicvariation:
+
+SystemicVariation
+@@@@@@@@@@@@@@@@@
+
+
+AbsoluteAbundance
+#################
+
+**Biological Definition**
+
+AbsoluteAbundance is the absolute and quantified amount of an entity
+within a genome, cell, or sample.
+
+**Computational Definition**
+
+AbsoluteAbundance is represented as a `subject`, which may be an
+Allele or Haplotype, or any object identifiable with a CURIE.
+
+**Information Model**
+
+.. list-table::
+   :class: reece-wrap
+   :header-rows: 1
+   :align: left
+   :widths: auto
+
+   * - Field
+     - Type
+     - Limits
+     - Description
+   * - _id
+     - :ref:`CURIE`
+     - 0..1
+     - Computed Identifier
+   * - type
+     - string
+     - 1..1
+     - MUST be "AbsoluteAbundance"
+   * - subject
+     - :ref:`Allele` | :ref:`Haplotype` | :ref:`CURIE`
+     - 1..1
+     - Subject of the relative abundance statement
+   * - amount
+     - :ref:`IntegerRange`
+     - 1..1
+     - At least one of `amount.min` or `amount.max` must be specified
+
+**Example**
+
+.. parsed-literal::
+
+    {
+      "amount": {
+        "max": 5,
+        "min": 0,
+        "type": "IntegerRange"
+      },
+      "subject": "ncbigene:1234",
+      "type": "AbsoluteAbundance"
+    }
+
+
+RelativeAbundance
+#################
+
+**Biological Definition**
+
+Relative abundance is the qualitative relative amount of a molecular
+species within a genome, cell, or sample.
+
+**Computational Definition**
+
+Relative abundance is represented as a combination of the `subject`
+and a qualitative relative amount.
+
+**Information Model**
+
+.. list-table::
+   :class: reece-wrap
+   :header-rows: 1
+   :align: left
+   :widths: auto
+
+   * - Field
+     - Type
+     - Limits
+     - Description
+   * - _id
+     - :ref:`CURIE`
+     - 0..1
+     - Computed Identifier
+   * - type
+     - string
+     - 1..1
+     - MUST be "RelativeAbundance"
+   * - subject
+     - :ref:`MolecularVariation` | :ref:`Gene`
+     - 1..1
+     - Subject of the relative abundance statement
+   * - amount
```

Comment:
I apologize if I am missing something, but doesn't the relativeAbundance need to have the ability to specify the base amount it is relative too? gt, lt, etc. need a reference amount to be relevant.

Reply:
You're not missing something, and it's as-intended.

We agreed to hold our nose that we're not specifying any reference. The whole point of the RelativeAbundance statement is to allow us to represent less-than-useful statements like "increased expression of HER2" from the real world. There's an implied and undefined normal.